tests/cases/conformance/jsx/tsxSpreadChildrenInvalidType.tsx

```diff
@@ -1,4 +1,5 @@
 // @jsx: react
+// @target: es6
```

Comment:
```suggestion
// @target: es2015,es5
```
Targeting both doesn't hurt and gives us a baseline for any future quirky regressions.

Reply:
Thanks! Done. :heavy_check_mark: